addons/website_slides_survey/views/website_slides_templates_lesson.xml

```diff
@@ -3,19 +3,19 @@
     <data>
         <template id="slide_content_detailed" inherit_id="website_slides.slide_content_detailed">
             <xpath expr="//div[hasclass('o_wslides_lesson_content_type')]" position="inside">
-                <t t-if="not certification_test_entry" t-set="begin_certification_label" t-value="'Begin certification'" />
-                <t t-if="certification_test_entry" t-set="begin_certification_label" t-value="'Test certification'" />
-                <div t-if="slide.slide_type == 'certification' and not certification_done" class="col mt32 mb8 d-flex justify-content-center">
+                <t t-if="slide.channel_id.is_member" t-set="begin_certification_label" t-value="'Begin certification'" />
```

Comment:
Not your fault, but while we are at it: the t-value will never be translated. 
Below, use a t-if/t-else and we don't need the t-att-title, not t-att-aria-label.